test/integration/create-resource/index.js

```diff
@@ -39,6 +41,20 @@ describe("", () => {
               expect(res.body.extra).to.be.undefined;
             });
 
+            let record;
+            it("should camelize attributes", (done) => {
+              mongoose.models.Organization.findById(createdId, (err, org) => {
```

Comment:
Hm. What if rather than calling mongoose directly here, we called public methods of the dbAdapter? Otherwise, we really can’t accurately test this in an integration test.